### PR TITLE
DP-25832: Fix accordion tests in patternlab

### PR DESCRIPTION
**Pull request by arthurbaghdas**
Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/develop/docs/for-developers/changelog-instructions.md)

## Related Issue / Ticket

- [JIRA issue](https://massgov.atlassian.net/browse/DP-25832)
- [Github issue]()

## Steps to Test

1.

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.

## Additional Notes:

Anything else to add?

#### Impacted Areas in Application

*

#### @TODO

*

### CODE REVIEW

File: packages/patternlab/styleguide/source/_patterns/02-molecules/action-step.twig [hunk 1]
```diff
@@ -3,10 +3,10 @@
 {% set link = actionStep.link %}
 
 <section
-  class="ma__action-step {{ accordion ? 'js-accordion ma__action-step--accordion' : '' }} {{ actionStep.isExpanded ? 'is-open':'' }}">
+  class="ma__action-step {{ accordion ? ' js-accordion ma__action-step--accordion ' : '' }} {{ actionStep.isExpanded ? 'is-open':'' }}">
```
Comment (clairesunstudio) [inline packages/patternlab/styleguide/source/_patterns/02-molecules/action-step.twig]:
```suggestion
  class="ma__action-step {{ accordion ? 'js-accordion ma__action-step--accordion' : '' }} {{ actionStep.isExpanded ? 'is-open':'' }}">

```

I don't think these spaces are necessary

Comment (clairesunstudio) [reply packages/patternlab/styleguide/source/_patterns/02-molecules/action-step.twig]: https://mayflower.digital.mass.gov/patternlab/b/develop/?p=molecules-action-step-as-accordion

Not seeing issues in develop

File: changelogs/DP-25832.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+#    component: Enter one or multiple (comma separated) valid component name(s) - see ___COMPONENT___ e.g. Color
+#    description: Describe the change and add the PR number. see ___DESCRIPTION___ e.g. Adds apples to apple trees for admin apple pickers (#PR)
+#    issue: Add a Jira ticket or issue number, e.g. DP-12345 or 124
+#    impact: Enter a valid impact, e.g. Minor
+
+#___EXAMPLE___
+Fixed:
+  - project: Patternlab
+    component: Accordions
+    description: Add Color stories and color gradients. (#909)
```
Comment (clairesunstudio) [inline changelogs/DP-25832.yml]:
```suggestion
    description: Set accordion initial state to collapse. (#1674)
```

File: packages/patternlab/styleguide/source/_patterns/02-molecules/action-step.twig (hunk 1 above)
Comment (arthurbaghdas) [reply packages/patternlab/styleguide/source/_patterns/02-molecules/action-step.twig]: I see the issue here https://mayflower.digital.mass.gov/patternlab/b/develop/?p=templates-location
Without speaces it was just concating values without spaces

File: changelogs/DP-25832.yml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    issue: DP-25832
+    impact: Minor
```
Comment (clairesunstudio) [inline changelogs/DP-25832.yml]:
```suggestion
    impact: Patch
```

File: packages/patternlab/styleguide/source/_patterns/02-molecules/action-step.twig [hunk 4]
```diff
@@ -3,10 +3,10 @@
 {% set link = actionStep.link %}
 
 <section
-  class="ma__action-step {{ accordion ? 'js-accordion ma__action-step--accordion' : '' }} {{ actionStep.isExpanded ? 'is-open':'' }}">
+  class="ma__action-step {{ accordion ? ' js-accordion ma__action-step--accordion ' : '' }} {{ actionStep.isExpanded ? 'is-open':'' }}">
   {% if accordion %}
     {% set headingLevel = actionStep.level ? : 3 %}
-    <h{{headingLevel}} class="ma__action-step__title {{ link ? 'ma__decorative-link' : ''}}">
+    <h{{headingLevel}} class="ma__action-step__title {{ link ? ' ma__decorative-link ' : ''}}">
```
Comment (clairesunstudio) [inline packages/patternlab/styleguide/source/_patterns/02-molecules/action-step.twig]:
```suggestion
    <h{{headingLevel}} class="ma__action-step__title {{ link ? 'ma__decorative-link' : ''}}">

```

File: packages/patternlab/styleguide/source/_patterns/02-molecules/action-step.twig [hunk 5]
```diff
@@ -50,7 +50,7 @@
   {% endif %}
 
   {% if actionStep.richText or actionStep.decorativeLink or actionStep.downloadLinks or actionStep.content %}
-    <div class="ma__action-step__content {{ accordion ? 'js-accordion-content' : '' }}">
+    <div class="ma__action-step__content {{ accordion ? ' js-accordion-content' : '' }}">
```
Comment (clairesunstudio) [inline packages/patternlab/styleguide/source/_patterns/02-molecules/action-step.twig]:
```suggestion
    <div class="ma__action-step__content {{ accordion ? 'js-accordion-content' : '' }}">

```

File: packages/patternlab/styleguide/source/_patterns/02-molecules/action-step.twig (hunk 1 above)
Comment (clairesunstudio) [reply packages/patternlab/styleguide/source/_patterns/02-molecules/action-step.twig]: really? Are you referring to the page alerts accordions? I'm not seeing concat issues 
Screen Shot 2022-09-13 at 9 52 38 AM
The spaces outside the double brackets should be sufficient. 

Also, the molecule seems okay too https://mayflower.digital.mass.gov/patternlab/b/develop/?p=templates-location

**Review by clairesunstudio — APPROVED** [review]

**Pull request merged**